CODEOWNERS

```
@@ -1,4 +1,9 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-* @MichaelSimons @mthalman @lbussell
+*               @dotnet-docker-reviewers
+
+monitor/        @dotnet-docker-reviewers @jander-msft
```

I suspect this reference is wrong - see https://github.com/dotnet/installer/blob/main/CODEOWNERS

```suggestion
monitor/        @dotnet/dotnet-docker-reviewers @jander-msft
```